code/game/objects/effects/decals/turfdecal/tilecoloring.dm

```diff
@@ -1,12 +1,25 @@
+GLOBAL_LIST_INIT(pride_month_colors, list("#fe0000","#fe9900","#ffff00","#009900","#0000fe","#cc0098","#fbfaf6","#feb0c8","#75d8ed", "#603814", "#000100"))
+GLOBAL_VAR_INIT(current_pride_color, 1)
+
 /obj/effect/turf_decal/tile
 	name = "tile decal"
 	icon_state = "tile_corner"
 	layer = TURF_PLATING_DECAL_LAYER
 	alpha = 110
+	var/pride_already_applied = FALSE
```

Comment:
[Please document your code to meet our code standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

Reply:
This variable is getting removed and deprecated.